check_pgactivity

```diff
@@ -6103,17 +6103,20 @@ sub check_replication_slots {
         SELECT slot.slot_name, slot.slot_type,
         COALESCE(
             floor(
-                (pg_xlog_location_diff(
-                     CASE WHEN pg_is_in_recovery()
-                        THEN pg_last_xlog_receive_location()
-                        ELSE pg_current_xlog_location()
-                      END
-                      , slot.restart_lsn)
-                -- this is needed to account for whole WAL retention and not
-                -- only size retention
-                + (pg_xlogfile_name_offset(restart_lsn)).file_offset
-                )
-                / (s.val)
+                CASE WHEN pg_is_in_recovery()
+                THEN (
+                  pg_xlog_location_diff(pg_last_xlog_receive_location(), slot.restart_lsn)
+                  -- this is needed to account for whole WAL retention and
+                  -- not only size retention
+                  + (pg_xlog_location_diff(restart_lsn, '0/0')%s.val)
```

Comment:
nitpicking, but could you add space around the %? For a second I thought this was a printf argument :)

Reply:
done :)